llama_index/embeddings/base.py

```diff
@@ -52,6 +52,7 @@ def __init__(
         self,
         embed_batch_size: int = DEFAULT_EMBED_BATCH_SIZE,
         tokenizer: Optional[Callable] = None,
+        **kwargs
```

Comment:
can you remove from base class?

Reply:
@jerryjliu  that would mean removing all params from OpenAI kwargs other than embed_batch_size and tokeniser.  Is it ok to handle kwargs like this in the OpenAIembedding constructor?

Reply:
Ok, have made the changes now.